src/spectral/rulesets/.defaultsForSpectral.yaml

```diff
@@ -149,3 +150,12 @@ rules:
     resolved: true
     then:
       function: error-response-schema
+  # ensure major version is in path
+  major-version-in-path:
+    description: 'All paths must contain the API major version as a distict path segment'
```

Comment:
Just a minor typo here with `distict` instead of `distinct`. Should be:

'All paths must contain the API major version as a distinct path segment'

Reply:
Fixed.